tools/devops/automation/templates/tests/stage.yml

```
@@ -74,6 +74,8 @@ stages:
   # we need to have the pkgs built and the device sets to be ran, that is decided via the labels or type of build during the build_packages stage
   condition: and(succeeded(), ${{ parameters.condition }})
 
+  variables:
+    WORKLOAD_VERSION: $[ stageDependencies.build_packages.build.outputs['workload_file.WorkloadVersion'] ] 
```

Part 3:

```suggestion
    IOS_WORKLOAD_VERSION: $[ stageDependencies.build_packages.build.outputs['workload_file.iOSWorkloadVersion'] ] 
```

needs to be expanded for all platforms.